modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/charon-config.xml.j2

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Remove this new line

Reply:
Fixed - https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/220